.github/workflows/ci.yml

```diff
@@ -0,0 +1,238 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+---
+name: Airflow
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+env:
+  BUILD_ID: ${{github.sha }}
+  MOUNT_LOCAL_SOURCES: "false"
+  MOUNT_HOST_AIRFLOW_VOLUME: "true"
+  FORCE_ANSWER_TO_QUESTIONS: "yes"
+  SKIP_CHECK_REMOTE_IMAGE: "true"
+  DB_RESET: "true"
+  VERBOSE: "true"
+  CI: "true"
+  # Should be a target branch
+  TRAVIS_BRANCH: "master"
+  TRAVIS: "true"
+jobs:
+  statics:
+    name: Static checks
+    runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: 3.6
+      AIRFLOW_MOUNT_SOURCE_DIR_FOR_STATIC_CHECKS: "true"
+      TRAVIS_JOB_NAME: "Static"
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: "Before install"
+        run: ./scripts/ci/ci_before_install.sh
+      - name: "Static checks"
+        if: success()
+        env:
+          PYTHON_VERSION: 3.6
+        run: ./scripts/ci/ci_run_all_static_checks.sh
+
+  statics-tests:
+    name: Pylint for tests
+    runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: 3.6
+      AIRFLOW_MOUNT_SOURCE_DIR_FOR_STATIC_CHECKS: "true"
+      TRAVIS_JOB_NAME: "Static"
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: "Before install"
+        run: ./scripts/ci/ci_before_install.sh
```

Comment:
Idea about getting rid of the unnecessary logs. ... I think in the current GA the "ci_before_install.sh" produces unnecessary logs that are not needed in 99% of cases.. And I think there is an  easy way to fix them: https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts

We could redirect both stdout and stderr (> file.txt 2>&1)  to a file and upload it as artifact of the build . This way we have access to it if needed but it will not clutter the log of the "real" build.